<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMAIII</class>
 <widget class="QMainWindow" name="OMAIII">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OMAIII - Data Adquisitor</string>
  </property>
  <widget class="QWidget" name="main">
   <widget class="QWidget" name="Plot" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>589</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QGroupBox" name="ScanSequence">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Sequence</string>
    </property>
    <widget class="QLabel" name="lbl_detectortemp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
     <property name="buddy">
      <cstring>le_detectortemp</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_detectortemp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_damode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_ignoredscans">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_scansnumber">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_exposuretime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_exposuretime">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure Time</string>
     </property>
     <property name="buddy">
      <cstring>le_exposuretime</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="lblscannumber">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NÂ° Scans</string>
     </property>
     <property name="buddy">
      <cstring>le_scansnumber</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_ignoredscans">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ignored Scans</string>
     </property>
     <property name="buddy">
      <cstring>le_ignoredscans</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_damode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DA Mode</string>
     </property>
     <property name="buddy">
      <cstring>le_damode</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AutoSave">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto Save</string>
    </property>
    <widget class="QLabel" name="lbl_basename">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Base Name</string>
     </property>
     <property name="buddy">
      <cstring>le_basename</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_initvalue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Init Value</string>
     </property>
     <property name="buddy">
      <cstring>le_initvalue</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_basename">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_initvalue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_autosave">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tlb_basename">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Scan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>181</width>
      <height>182</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Scan</string>
    </property>
    <widget class="QPushButton" name="btn_run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_scannumbers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_scannumbers">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scans Loop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>117</height>
     </rect>
    </property>
    <property name="title">
     <string>Monocromathor</string>
    </property>
    <widget class="QLabel" name="lbl_red">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="buddy">
      <cstring>le_red</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_counter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>39</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Counter</string>
     </property>
     <property name="buddy">
      <cstring>le_counter</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_red">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_counter">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Background">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Background</string>
    </property>
    <widget class="QPushButton" name="btn_sustractbkg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sustract</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_selectedbkg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="tlb_selectbkg">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_autobkgsus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Background Sustraction</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSort"/>
   </widget>
   <widget class="QMenu" name="menuOMA">
    <property name="title">
     <string>OMA</string>
    </property>
    <addaction name="actionSetup"/>
   </widget>
   <widget class="QMenu" name="menuPloter">
    <property name="title">
     <string>Ploter</string>
    </property>
    <addaction name="actionIniciar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOMA"/>
   <addaction name="menuPloter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Setup RS232</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort...</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_exposuretime</tabstop>
  <tabstop>le_scansnumber</tabstop>
  <tabstop>le_ignoredscans</tabstop>
  <tabstop>le_damode</tabstop>
  <tabstop>le_detectortemp</tabstop>
  <tabstop>btn_update</tabstop>
  <tabstop>btn_run</tabstop>
  <tabstop>btn_stop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
